examples/Python/ChatApp/README.md

```diff
@@ -0,0 +1,56 @@
+# Azure App Configuration - Python ChatApp Sample
+
+This sample demonstrates using Azure App Configuration to configure Azure OpenAI settings for a chat application built with Python.
+
+## Features
+
+- Integrates with Azure OpenAI for chat completions
+- Dynamically refreshes configuration from Azure App Configuration
+
+## Prerequisites
+
+- Python 3.8 or later
+- An Azure subscription with access to:
+  - Azure App Configuration service
+  - Azure OpenAI service
+- Required environment variables:
+  - `AZURE_APPCONFIG_ENDPOINT`: URL of your Azure App Configuration instance
+  - `AZURE_OPENAI_API_KEY`: API key for Azure OpenAI (optional if using DefaultAzureCredential)
+
+## Setup
+
+1. Clone the repository
+2. Install the required packages:
+   ```bash
+   pip install -r requirements.txt
+   ```
+3. Configure your Azure App Configuration store with these settings:
+   ```
+   ChatApp:AzureOpenAI:Endpoint - Your Azure OpenAI endpoint URL
+   ChatApp:AzureOpenAI:DeploymentName - Your Azure OpenAI deployment name
+   ChatApp:Model - An AI configuration entry containing the following settings:
```

Comment:
This is a json object with the content type of application/json; profile=\"https://azconfig.io/mime-profiles/ai/chat-completion\"